shared/uavobjectdefinition/loggingsettings.xml

```diff
@@ -1,6 +1,7 @@
 <xml>
 	<object name="LoggingSettings" singleinstance="true" settings="true">
 		<description>Settings for the logging module</description>
+		<field name="LogDestination" units="" type="enum" options="SPIFlash,OpenLog" elements="1" defaultvalue="OpenLog"/>
```

Comment:
Any board can log to the OpenLog, but only some boards have the SPI Flash chip, so OpenLog seemed to be the logical choice for a default.

Which board crashes?  I haven't seen that but would like to try and replicate it.

This could be setup such that if a port is assigned to the OpenLog, it automatically determines that it should log to the device, doing away with the LogDestination object.  Looking into that and also removing the destination tie into PIOS_HAL.c.